pkg/utils/webhook/webhook.go

```diff
@@ -72,6 +73,19 @@ func (iv *InstallationValidator) Handle(ctx context.Context, req admission.Reque
 	logger := iv.log.WithValues(lc.KeyResourceGroup, req.Kind.Group, lc.KeyResourceKind, req.Kind.Kind, lc.KeyResourceVersion, req.Kind.Version)
 	ctx = logging.NewContext(ctx, logger)
 
+	timeBefore := time.Now()
+	result := iv.handlePrivate(ctx, req)
+	timeAfter := time.Now()
+
+	if timeAfter.Sub(timeBefore) > 9*time.Second {
+		logger.Info("validation request required more than 9 seconds")
```

Comment:
Can you please also log the duration?
Why 9 seconds? I think this needs at least a comment in code explaining this otherwise the knowledge will be forgotten in the future.